variables.tf

```diff
@@ -19,7 +19,11 @@ variable "family" {
   type        = string
   description = "The family of the DB parameter group."
 }
-
+variable "db_security_group_name" {
+  type        = string
+  description = "Optional name of the security group to be created for the RDS instance. Defaults to {var.identifier}-db_sg"
+  default     = "db_sg"
```

Comment:
(nit) in most terraform modules I've made this default to `null` and then I would check if `var.db_security_group_name != null` I would use the variable, else the generated default one. I personally don't like setting a default that actually isn't used mainly because it's a little misleading, but again this is more a personal preference than anything.